syntaxes/dancer.json

```diff
@@ -15,7 +15,7 @@
         "dancer": {
             "patterns": [
                 {
-                    "match": "^\\s*\\b(hook)\\s+(\\w+)\\s+(?==>\\s*sub\\b)",
```

Comment:
This would no longer recognize:
```perl
hook before => sub {
```
All the examples I saw in the Dancer manual used `=>` in them. Although if it's optional, perhaps you could match the regex from pltags? `(?:=>)?`